src/components/MentionTextArea/index.tsx

```diff
@@ -121,7 +121,10 @@ const MentionTextArea = ({
         }
       },
       extensions: [
-        StarterKit,
+        StarterKit.configure({
+          bulletList: false,
+          orderedList: false
+        }),
```

Comment:
The TipTap extension documents the [possible nodes and marks](https://tiptap.dev/docs/editor/extensions/functionality/starterkit) that can be configured -- should we opt to disable more than what we have here?

```suggestion
        StarterKit.configure({
          // nodes
          blockquote: false,
          bulletList: false,
          codeBlock: false,
          document: false,
          hardBreak: true,
          heading: false,
          horizontalRule: false,
          listItem: false,
          orderedList: false,
          paragraph: true,
          text: true,
          // marks
          bold: false,
          code: false,
          italic: false,
          strike: false,
        }),
```

This might not work exactly as-is, but just thinking that it might be good to be explicit.

I guess maybe since the `StarterKit` contains these extensions, maybe we could just enable only the extensions we actually care about instead of `StarterKit minus a bunch of things`? I think each of those nodes/marks are individual extensions that we could import instead of `StarterKit`

Reply:
I think if we wanted to just pull the few things we cared about from that page, we could probably just grab:
- [Text](https://tiptap.dev/docs/editor/extensions/nodes/text)
- [Paragraph](https://tiptap.dev/docs/editor/extensions/nodes/paragraph)
- [HardBreak](https://tiptap.dev/docs/editor/extensions/nodes/hard-break) (maybe?)

Reply:
i skipped HardBreak, that one allows `<br>` tags. TipTap can instead stack `<p>` tags for line breaks, which i would guess is preferred